brion/spikeReport.h

```diff
@@ -62,7 +62,7 @@ class SpikeReport : public boost::noncopyable
      * State::ended means that the report end is reached (in read mode)
      * State::failed means that a error occured when readyn or writing the report
      */
-    enum class State : uint8_t
+    enum class State : uint32_t
```

Comment:
What's the reason for using uint32_t? (in this case I think you can even omit it)